tcms/core/middleware.py

```diff
@@ -1,6 +1,8 @@
+# pylint: disable=no-self-use, too-few-public-methods
+
 from django.utils.deprecation import MiddlewareMixin
```

Comment:
Can you open a new issue for this middleware. It looks like the base class will be deprecated soon and honestly this middleware looks like it should be removed because it negates the CSRF mechanism in Django.

Reply:
Done. #297